src/commands/index.ts

```diff
@@ -248,3 +248,4 @@ import './rotate'
 import './search'
 import './select'
 import './selectObject'
+import './goto'

```

Comment:
That's the line I forgot! Can you put it higher so that imports are alphabetically ordered please?